components/bitcoin-topics/Navigation.tsx

```diff
@@ -11,15 +11,24 @@ import {
 import { Topic, allTopics } from "@/.contentlayer/generated"
 import { useState, useEffect, useCallback, useMemo } from "react"
 import { motion } from "framer-motion"
-import { FaChevronRight, FaChevronDown } from "react-icons/fa"
+import {
+    FaChevronRight,
+    FaChevronDown,
+    FaCheck,
```

Comment:
Never used

Reply:
hey @jrakibi this is still unused.
We can tighten up `eslint` so it catches stuff like this (in another PR)